targets/targets.json

```diff
@@ -6802,6 +6802,62 @@
         "release_versions": ["5"],
         "detect_code": ["3703"]
     },
+    "FAMILY_Apollo3": {
+        "inherits": ["Target"],
+        "core": "Cortex-M4F",
+        "features": ["BLE"],
+        "default_toolchain": "GCC_ARM",
+        "supported_toolchains": ["GCC_ARM", "ARMC6"],
+        "public": false,
+        "extra_labels": ["Ambiq_Micro", "Apollo3", "CORDIO"],
```

Comment:
please follow the style (all these should be on a new line) - same as device_has is below

Reply:
done